client/shipping-label/state/actions.js

```diff
@@ -143,6 +161,17 @@ export const openPrintingFlow = () => ( dispatch, getState, { storeOptions, addr
 			expandFirstErroneousStep( dispatch, getState, storeOptions );
 		};
 
+		const showPreviewAfterRateFetch = () => {
+			const newForm = getState().shippingLabel.form;
+			const newAvailableRates = newForm.rates.available;
+			const allRatesPreselected = _.every( newAvailableRates, ( availableRate ) => ( _.some( availableRate.rates, 'is_selected' ) ) );
+			if ( allRatesPreselected ) {
```

Comment:
Is all this logic necessary considering the checks internal to `refreshPreview()`?

Could we just simply call `refreshPreview()` and `expandStepAfterAction()` here?